### PR TITLE
copy improvements suggested by translator for homepage and features page

### PR DESCRIPTION
**Pull request by jutta-frieden**
Our translator did an audit of our German sites and we have a list of improvements which we are adding to the marketing pages. This is for homepage and the features page:
https://gocardless.com/de-de/
https://gocardless.com/de-de/funktionen/

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@georg911 or @a-xin Could you have a quick look through the new (nicer) German?

screen shot 2016-10-18 at 17 44 47
screen shot 2016-10-18 at 17 44 56

screen shot 2016-10-18 at 17 43 27
screen shot 2016-10-18 at 17 43 39
screen shot 2016-10-18 at 17 43 46
screen shot 2016-10-18 at 17 44 02

File: app/pages/features/features.de.js [hunk 1]
```diff
@@ -184,20 +184,20 @@ export default class FeaturesDe extends React.Component {
...
                   <p className='u-size-5of6 u-color-dark-gray u-margin-Bm'>
-                    Unser übersichtliches Online Dashboard sowie unsere Email Benachrichtigungen halten Sie jederzeit
-                    auf dem Laufenden über Ihre Kunden und deren Zahlungen.
+                    Unser übersichtliches Online-Dashboard sowie unsere E-Mail-Benachrichtigungen halten
+                    Sie jederzeit über Ihre Kunden und deren Zahlungen auf dem Laufenden.
                   </p>
                   <h3 className='u-text-xs'>Exportieren Sie Ihre Daten</h3>
                   <p className='u-size-5of6 u-color-dark-gray u-margin-Bm'>
-                    Sehen Sie Ihre Daten genau so, wie Sie das möchten. GoCardless ist mit den meisten Verwaltungssoftwares kompatibel.
+                    Managen Sie Ihre Daten genau nach Ihren Vorstellungen. GoCardless ist mit den meisten Verwaltungssoftwares kompatibel.
```
Comment (a-xin) [inline app/pages/features/features.de.js]: Verwaltungsprogrammen?

File: app/pages/features/features.de.js [hunk 2]
```diff
@@ -139,11 +139,11 @@ export default class FeaturesDe extends React.Component {
...
                     </figure>
                     <div className='u-text-heading u-color-dark-gray u-text-light u-text-no-smoothing u-text-s u-margin-Tm'>
-                      Starke Verbleibquote
+                      Schaffen Sie langfristige Kundenbeziehungen
                     </div>
                     <p className='u-color-dark-gray u-margin-Txs'>
-                      Da GoCardless auf dem Lastschriftverfahren aufbaut,
-                      verlieren Sie weniger Kunden durch Kreditkartenablauf oder Kündigungen.
+                      Dank der Verwendung des Lastschriftverfahrens können Sie mit GoCardless keine Kunden durch
+                      abgelaufene oder gesperrte Kreditkarten verlieren.
```
Comment (a-xin) [inline app/pages/features/features.de.js]: `Dank der Verwendung des Lastschriftverfahrens verlieren Sie mit GoCardless keine Kunden durch abgelaufene oder gesperrte Kreditkarten.` ?

File: app/pages/features/features.de.js (hunk 1 above)
Comment (jutta-frieden) [inline app/pages/features/features.de.js]: Changed.

File: app/pages/features/features.de.js (hunk 2 above)
Comment (jutta-frieden) [inline app/pages/features/features.de.js]: Changed. Good spot!

**Comment by jutta-frieden** [comment]
Great comments, thanks @a-xin 👍

**Pull request merged**